### PR TITLE
SO-83

### PR DESCRIPTION
**Pull request by JDoan03**
SO-83:
Edit organization button
- Update organization name
- Update organization image

### CODE REVIEW

File: src/components/Organization/EditOrganizationModal/EditOrgModal.tsx
Comment (hashemJaber) [inline src/components/Organization/EditOrganizationModal/EditOrgModal.tsx:1]: Really really clean code Joseph, nice work, clean way to do handleChange, good job

File: src/components/SharedComponents/DetailsCard/DetailsCard.tsx
Comment (hashemJaber) [inline src/components/SharedComponents/DetailsCard/DetailsCard.tsx:1]: I see you made changes to the logic of editing a organization and made it cleaner, again good job

File: src/types/componentTypes.ts
Comment (hashemJaber) [inline src/types/componentTypes.ts:1]: Good thinking

**Review by hashemJaber — APPROVED** [review]
Good Job indeed 
> [!NOTE]
> Approved in my book

File: src/components/Organization/EditOrganizationModal/EditOrgModal.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+		setImage(e.target.files[0]);
+		setImageURL(URL.createObjectURL(e.target.files[0]));
+	};
+
+	const handleSubmit = async (e: any) => {
+		// upload image
+		await editOrganization(e);
+	};
+
+	const editOrganization = async (e: any) => {
```
Comment (jvniorrr) [inline src/components/Organization/EditOrganizationModal/EditOrgModal.tsx:57]: Once implemented to main branch, lets move such functions to a separate dedicated file under the `lib/actions` folder likely, and subfolder if you want giving it whatever name you find fitting.

File: src/components/Organization/EditOrganizationModal/EditOrgModal.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+	const [formData, setFormData] = useState({
+		orgName: organization.name,
+		orgImageUrl: organization.image,
+		orgImage: useState<any>([]),
+	});
+
+	const DEFAULT_IMAGE =
+		'https://apqmqmysgnkmkyesdrnn.supabase.co/storage/v1/object/public/profile-avatars/wyncoservices.png';
+	const PREVIOUS_IMAGE = organization.image;
+	console.log(PREVIOUS_IMAGE);
```
Comment (jvniorrr) [inline src/components/Organization/EditOrganizationModal/EditOrgModal.tsx:27]: delete line

File: src/components/Organization/EditOrganizationModal/EditOrgModal.css
Comment (jvniorrr) [inline src/components/Organization/EditOrganizationModal/EditOrgModal.css:1]: Awesome

**Review by jvniorrr — APPROVED** [review]
Testing will be needed and just removing few lines. Thank you for great work.

**Pull request merged**